20971 add notes about relaxed range restrictions on `formatRange` and `selectRange` (#21011)
mdn/content: files/en-us/mozilla/firefox/releases/105/index.md

### JavaScript

> - Range restrictions are relaxed on `formatRange` and `selectRange` functions for [`Intl.DateTimeFormat`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat), [`Intl.NumberFormat`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat), and [`Intl.PluralRules`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/PluralRules).
>   For example, `formatRange(x, y)` can be called where and `x` > `y` such as `formatRange(-500, -1000)`.
>   The intention of this change is to relax ordering rules so that negative ranges (e.g., "losses ranging from -$500 to -$1000") can be formatted correctly ({{bug(1780545)}}).

Reviewer: I think the examples and specific details can be covered in the `formatRange` and `selectRange` pages (probably separate PR). 

```suggestion
- Range restrictions have been relaxed on `formatRange` and `selectRange` functions for [`Intl.DateTimeFormat`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat), [`Intl.NumberFormat`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat), and [`Intl.PluralRules`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/PluralRules) objects. This change now allows negative ranges ({{bug(1780545)}}).
```

Author: Sounds good to me

### APIs

#### DOM

- The [TextDecoderStream](/en-US/docs/Web/API/TextDecoderStream) and [TextEncoderStream](/en-US/docs/Web/API/TextEncoderStream) interfaces, part of the [Encoding API](/en-US/docs/Web/API/Encoding_API), are now supported ({{bug(1486949)}}).

> - The [OffscreenCanvas API](/en-US/docs/Web/API/OffscreenCanvas) provides a canvas that can be rendered off-screen in both Window and [Web Worker contexts](/en-US/docs/Web/API/OffscreenCanvas#asynchronous_display_of_frames_produced_by_an_offscreencanvas), decoupling the DOM and the Canvas API so that `<canvas>` elements are no longer dependent on the DOM.
>   The [OffscreenCanvasRenderingContext2D](/en-US/docs/Web/API/OffscreenCanvasRenderingContext2D) interface is supported and enabled by default ({{bug(1746110)}}).

Reviewer: ```suggestion
- The [OffscreenCanvas](/en-US/docs/Web/API/OffscreenCanvas) API now provides a canvas that can be rendered off-screen in both window and [web worker](/en-US/docs/Web/API/OffscreenCanvas#asynchronous_display_of_frames_produced_by_an_offscreencanvas) contexts, decoupling the DOM and the Canvas API. This allows `<canvas>` elements to no longer be dependent on the DOM.
- The [OffscreenCanvasRenderingContext2D](/en-US/docs/Web/API/OffscreenCanvasRenderingContext2D) interface is now supported and enabled by default ({{bug(1746110)}}).
```
- Please check if the lower casing suggestions are correct
- Added suggestion to split the long sentence into two
- Should 'OffscreenCanvasRenderingContext2D' be a separate point?

Author: Casing suggestions look good. I'd lean towards keeping it in one point as the `OffscreenCanvasRenderingContext2D` is what made up most of the implementation work for this as a whole. What do you think?

Author: Added improvements in https://github.com/mdn/content/pull/21011/commits/99794438270ea412860e1cff5c1eb7ffd9d15937
